docker/worker/worker.py

```diff
@@ -472,6 +472,9 @@ def _do_update(self, source_repo, repo, vulnerability, relative_path,
           timestamp=osv.utcnow(),
           status=osv.BugStatus.PROCESSED,
           source_of_truth=osv.SourceOfTruth.SOURCE_REPO)
+    elif bug.import_last_modified == vulnerability.import_last_modified:
```

Comment:
Rather than add this complexity here, could we do this in the importer instead? It's better to keep this logic closer to the importer as that's where most of the decisions are being made about what to pocess.  

We're already downloading entries prior to requesting workers to process them:.

https://github.com/google/osv.dev/blob/57ac34b2f15a80c584a8fa02a50d16e6fc4e10fc/docker/importer/importer.py#L303

So we just need to extend this part to parse out the `modified` field for a comparison.

Reply:
Done, moved the check into importer.